src/test/java/org/sagebionetworks/bridge/exporter/handler/SynapseExportHandlerTest.java

```diff
@@ -499,13 +501,15 @@ public static Item makeDdbRecord() {
         return new Item().withLong("createdOn", DUMMY_CREATED_ON).withString("healthCode", DUMMY_HEALTH_CODE)
                 .withString("id", DUMMY_RECORD_ID).withString("metadata", DUMMY_METADATA_JSON_TEXT)
                 .withStringSet("userDataGroups", DUMMY_DATA_GROUPS)
+                .withString("userSubstudyMemberships", DUMMY_SUBSTUDY_MEMBERSHIPS)
```

Comment:
I think withString is now no longer needed.

Reply:
Removed